lib/WP_Auth0_Api_Client.php

```diff
@@ -51,12 +51,8 @@ public static function get_connect_info( $opt = '' ) {
 				'client_secret_encoded' => $a0_options->get( 'client_secret_b64_encoded' ),
 				'connection' => $a0_options->get( 'db_connection_name' ),
 				'app_token' => $a0_options->get( 'auth0_app_token' ),
-				'audience' => $a0_options->get( 'api_audience' ),
+				'audience' => self::get_endpoint( 'api/v2/' ),
```

Comment:
This is for the Management API and the audience for that can't be changed so no reason to have it be editable